tests/unit/test_wheels.py

```diff
@@ -21,9 +21,10 @@ def test_build_and_upload_wheel():
 
     wheels = WheelsV2(installation, product_info)
     with wheels:
-        assert os.path.exists(wheels._local_wheel)
+        for wheel in wheels._local_wheel:
```

Comment:
it's prohibited to modify existing unit tests, because they guard Public API of this library. add new tests to work with this new increment.